packages/dataviews/src/components/dataviews-pagination/index.tsx

```diff
@@ -49,12 +49,7 @@ function DataViewsPagination() {
 	return (
 		!! totalItems &&
 		totalPages !== 1 && (
-			<HStack
-				expanded={ false }
-				spacing={ 6 }
-				justify="end"
-				className="dataviews-pagination"
-			>
+			<HStack expanded={ false } justify="end">
```

Comment:
I think the `.dataviews-pagination` class should remain, otherwise BEM-style selectors won't have a parent group, like `dataviews-pagination__page-select`.

Instead of leaving the CSS class, how about deleting all the styles [here](https://github.com/WordPress/gutenberg/blob/01c2c6545cd6efd00da0f63b8e5809a3eb3bf6da/packages/dataviews/src/components/dataviews-pagination/style.scss#L1-L12)?

Reply:
I think meanwhile the class was readded.